pyproject.toml

```diff
@@ -41,14 +41,6 @@ Documentation = "https://bioio-devs.github.io/bioio"
 # https://peps.python.org/pep-0621/#dependencies-optional-dependencies
 [project.optional-dependencies]
 lint = [
-  "black>=22.3.0",
-  "check-manifest>=0.48",
```

Comment:
`check-manifest` wasn't being used in `pre-commit`, but instead it was installed without a specific version before `just install` was ran [here](https://github.com/bioio-devs/bioio/blob/main/.github/workflows/ci.yml#L32) so it seems safe to delete unless I am missing a usage.

Reply:
The only reason I include black is that VSCode will auto format on save when it is installed.

(I may be wrong about that, that is how I have set it up anyway)

Check manifest can go. It's very useful but used infrequently.

Reply:
Similarly, does VSCode not need any of these installed to have all the nice linting, formatting, type hinting, etc?